domains/jthweb.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "is-a-dev",
```

"username" should ne you Github username
```suggestion
    "username": "jthweb",
```